content/gherkin/reference.md

```diff
@@ -330,6 +331,60 @@ to match the step against a step definition.
 
 You can also use parameters in [multiline step arguments](#step-arguments).
 
+## Rule
+
+The `Rule` entity is a rather new idiom in Gherkin (since Gherkin v6).
+Each `Rule` is intended to represent one *business rule* that should be implemented.
+A `Rule` is used to group together several `Example` (aka: `Scenario`) / `Scenario Outline`
```

Comment:
Could you replace "`Example` (aka: `Scenario`) / `Scenario Outline`" with "scenarios" - for readability.

Reply:
Yes, I could. 

But you have no heading on this page that says "Scenario", only "Example" because it is the new, preferred keyword (as I understand it from the rest of the description on this page).
Therefore, the docs would be no longer self-explaining for a beginner in my opinion.

@mlvandijk 
Just decide what you think is best (and I make the change).